keps/sig-scheduling/5278-nominated-node-name-for-expectation/README.md

```diff
@@ -205,21 +205,22 @@ based on the expected pod placement.
 
 ### External components want to specify a preferred pod placement
 
-The cluster autoscaler or Kueue internally calculates the pod placement,
-and create new nodes or un-gate pods based on the calculation result. 
+The ClusterAutoscaler or Karpenter internally calculate the pod placement,
```

Comment:
@sanposhiho - there are multiple changes including:

- small updates to motivation and user stories
- rewrite of risks section based on the end state we propose 
- discussion of new risk
- addressing some comments discussed on the original PR

I think that you should take adjustments from my commit that you think make sense and just incorporate them to your PR and I will close this one after that.